ApplicationPattern.yaml

```diff
@@ -8087,6 +8163,246 @@ paths:
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - response-profile-1-0:datatype
+                properties:
+                  response-profile-1-0:datatype:
+                    type: string
+                    example: 'string'
+        '400':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '401':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '403':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '404':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '500':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        default:
+          $ref: '#/components/responses/responseForErroredOamRequests'
+  /core-model-1-4:control-construct/profile-collection/profile={uuid}/response-profile-1-0:response-profile-pac/response-profile-configuration/value:
+    parameters:
+      - in: path
+        name: uuid
+        required: true
+        schema:
+          type: string
+          pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-response-p-([0-9]{3})$'
+          example: 'ro-2-0-0-response-p-000'
+    get:
+      operationId: getGenericResponseProfileValue
+      summary: 'Returns the Value of the Field'
+      tags:
+        - GenericResponseProfile
+      security:
+        - basicAuth: []
+      responses:
+        '200':
+          description: 'Value of the field provided'
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - response-profile-1-0:value
+                properties:
+                  response-profile-1-0:value:
+                    type: string
+                    example: 'ownApplicationName'
+        '400':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '401':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '403':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '404':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '500':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        default:
+          $ref: '#/components/responses/responseForErroredOamRequests'
+    put:
+      operationId: putGenericResponseProfileValue
+      summary: 'Configures the Value of the Field'
+      tags:
+        - ActionProfile
+      security:
+        - basicAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - response-profile-1-0:value
+              properties:
+                response-profile-1-0:value:
+                  type: string
+                  example: 'ownApplicationName'
```

Comment:
It shall contain the value as it shall be written into the referenced field.
If the referenced field shall contain a reference, then a reference to be represented here.
Wrote a 'to' instead of a 'from' to indicate the direction.